files/image_config/interfaces/interfaces-config.sh

```diff
@@ -12,6 +12,12 @@ fi
 
 sonic-cfggen -d -a '{"hwaddr":"'$SYSTEM_MAC_ADDRESS'"}' -t /usr/share/sonic/templates/interfaces.j2 > /etc/network/interfaces
 
+NULL_STRING=$(redis-cli -n 4 hget SWITCH\|SWITCH_ATTR switch_mac_null_string)
+CFGDB_SWITCH_MAC=$(redis-cli -n 4 hget SWITCH\|SWITCH_ATTR switch_mac)
+if [ "$CFGDB_SWITCH_MAC" == "$NULL_STRING" ]; then
+    redis-cli -n 4 hset SWITCH\|SWITCH_ATTR switch_mac $SYSTEM_MAC_ADDRESS
```

Comment:
I suggest just to write to SWITCH_MAC using the SYSTEM_MAC. The reason is that if we need to use another mac, we need to also change orchagent to use that mac. I suggest to change that via a separate CR.

Reply:
done